setup.py

```diff
@@ -92,7 +92,7 @@ def run(self):
                 logger.info('copying %s -> %s', source, dest)
                 copyfile(source, dest)
                 result = subprocess.run(['patchelf', '--print-needed', dest], check=True, stdout=subprocess.PIPE, universal_newlines=True)
-                cuda_dependencies = ['libcublas.so', 'libcudnn.so', 'libcudart.so']
+                cuda_dependencies = ['libcublas.so', 'libcudnn.so', 'libcudart.so', 'libcurand.so']
```

Comment:
A better way to define a list is this:
```
cuda_dependencies = [
    'libcublas.so',
    'libcudnn.so',
    'libcudart.so',
    'libcurand.so',
]
```
This would serve two purposes:
1) If you have to add a new shared object to this list, you won't need to modify the existing code.
2) PEP8 recommends a maximum line length of 79 characters, which we would easily be able to follow.

Reply:
Thank you! As today is the code freeze deadline, I'll do it later.